src/algorithms/wcc.c

```
@@ -0,0 +1,192 @@
+	ASSERT(components != NULL);
+
+	ASSERT( (lbls != NULL && n_lbls > 0) || (lbls == NULL && n_lbls == 0) );
+	ASSERT( (rels != NULL && n_rels > 0) || (rels == NULL && n_rels == 0) );
+
+	// nullify outputs
+	*N          = NULL;
+	*components = NULL;
+
+	// build matrix on which we'll compute WCC
+	GrB_Matrix A = _Build_Matrix(g, N, lbls, n_lbls, rels, n_rels);
+	ASSERT(A  != NULL);
```

Cppcheck (2.10-2)

[error] Null pointer dereference

(nullPointer)